kubevirt/intro.md

```diff
@@ -0,0 +1 @@
+This scenario demonstrates how kubevirt can be used to start a virtual machine
```

Comment:
expanded to:

This scenario demonstrates how KubeVirt can be used to start a virtual machine

KubeVirt is a virtualization management add-on for Kubernetes.
It allows a user to run virtual machines on Kubernetes just like pods.
Because KubeVirt is integrating with Kubernetes API and resource model, the
same set of tools can be used to manage and monitor virtual machines.
 

In the current pr, is this not enough?

Reply:
A user that gets into the katacoda scenario may not know anything about KubeVirt, we should introduce it in a first sentence, like:

> KubeVirt is a virtual machine management add-on for Kubernetes. The aim is to provide a common ground for virtualization solutions on top of Kubernetes.
And then continue with the: This scenario ...

Reply:
done in next pr